2345/3/models/comments_rating.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsRating
+  LIMIT = 50
+
+  def initialize(comments)
+    @comments = comments
+  end
+
+  # :reek:FeatureEnvy
```

Comment:
this is bad practice. 
It means that code get too complex
I could show how to fix it - but there are no specs for the class - so I will take much time for me (other developer) to understand what it does and refactor or extend.

If you want -  you could add specs for this particular class and I'll show how to fix the  'FeatureEnvy' issue
or you can fix it yourself